enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -31,23 +31,35 @@ import {
   type InteractiveQuestionDefaultViewProps,
 } from "embedding-sdk/components/private/InteractiveQuestionDefaultView";
 import { withPublicComponentWrapper } from "embedding-sdk/components/private/PublicComponentWrapper";
-import type { SdkQuestionId } from "embedding-sdk/types/question";
+import type { InteractiveQuestionQuestionIdProps } from "embedding-sdk/components/public/InteractiveQuestion/types";
 
-export type InteractiveQuestionProps = PropsWithChildren<{
-  questionId: SdkQuestionId;
-  plugins?: InteractiveQuestionProviderProps["componentPlugins"];
-}> &
-  Pick<
-    InteractiveQuestionProviderProps,
-    | "questionId"
-    | "onBeforeSave"
-    | "onSave"
-    | "entityTypeFilter"
-    | "isSaveEnabled"
-    | "initialSqlParameters"
-    | "withDownloads"
-    | "targetCollection"
-  >;
+/**
+ * @hidden
```

Comment:
Is this the only type excluded from the doc? I only see this one so far, I feel like there should be more base interfaces that are used to construct other interfaces and types.

Reply:
We will adjust types more deeply in M2 of this project

Reply:
Removed export for this type, now it is the `internal` type in generated docs